### PR TITLE
Issue735 data center chiller wse

### PR DESCRIPTION
**Pull request by YangyangFu**

### CODE REVIEW

File: Buildings/ChillerWSE/BaseClasses/Examples/BaseClasses/PartialPlantWithControl.mo [hunk 1]
```diff
@@ -0,0 +1,85 @@
+within Buildings.ChillerWSE.BaseClasses.Examples.BaseClasses;
+partial model PartialPlantWithControl
+  "Partial model that tests source plant with temperature control"
+  import Buildings;
```
Comment (JayHuLBL) [inline Buildings/ChillerWSE/BaseClasses/Examples/BaseClasses/PartialPlantWithControl.mo]: delete `import Buildings;`

File: Buildings/ChillerWSE/BaseClasses/Examples/BaseClasses/PartialPlantWithControl.mo [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+  parameter Modelica.SIunits.Pressure dp1_nominal=60000
+    "Nominal pressure difference on medium 1 side";
+  parameter Modelica.SIunits.Pressure dp2_nominal=60000
+    "Nominal pressure difference on medium 2 side";
+
+  Fluid.Sources.MassFlowSource_T           sou1(
+    redeclare package Medium = Medium1,
+    use_T_in=true,
+    m_flow=m1_flow_nominal,
+    T=298.15)
```
Comment (JayHuLBL) [inline Buildings/ChillerWSE/BaseClasses/Examples/BaseClasses/PartialPlantWithControl.mo]: add comment

File: Buildings/ChillerWSE/BaseClasses/Examples/BaseClasses/PartialPlantWithControl.mo [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+  Modelica.Blocks.Sources.Step T1_in(
+    height=2,
+    offset=273.15 + 5,
+    startTime=300)    "Condenser inlet temperature"
+    annotation (Placement(transformation(extent={{-100,20},{-80,40}})));
+  Fluid.Sources.MassFlowSource_T           sou2(
+    redeclare package Medium = Medium2,
+    use_T_in=true,
+    m_flow=m2_flow_nominal,
+    T=291.15)
```
Comment (JayHuLBL) [inline Buildings/ChillerWSE/BaseClasses/Examples/BaseClasses/PartialPlantWithControl.mo]: comment

File: Buildings/ChillerWSE/BaseClasses/Examples/BaseClasses/PartialPlantWithControl.mo [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+    annotation (Placement(transformation(extent={{80,-20},{60,0}})));
+  Modelica.Blocks.Sources.Constant
+                               T2_in(k=273.15 + 15)
+                   "Evaporator inlet temperature"
+    annotation (Placement(transformation(extent={{58,-60},{78,-40}})));
+  Fluid.Sources.FixedBoundary           sin1(
+    redeclare package Medium = Medium1,
+    nPorts=1)                           annotation (Placement(
```
Comment (JayHuLBL) [inline Buildings/ChillerWSE/BaseClasses/Examples/BaseClasses/PartialPlantWithControl.mo]: move annotation to new line

File: Buildings/ChillerWSE/BaseClasses/Examples/BaseClasses/PartialPlantWithControl.mo [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+      points={{79,-50},{96,-50},{96,-6},{82,-6}},
+      color={0,0,127},
+      smooth=Smooth.None));
+  connect(res2.port_b,sin2. ports[1]) annotation (Line(
+      points={{-72,-20},{-80,-20}},
+      color={0,127,255},
+      smooth=Smooth.None));
+  connect(res1.port_b, sin1.ports[1])
+    annotation (Line(points={{60,48},{80,48}},         color={0,127,255}));
+  annotation (Icon(coordinateSystem(preserveAspectRatio=false)), Diagram(
```
Comment (JayHuLBL) [inline Buildings/ChillerWSE/BaseClasses/Examples/BaseClasses/PartialPlantWithControl.mo]: add info section

File: Buildings/ChillerWSE/BaseClasses/Examples/BaseClasses/PartialPlantWithControl.mo [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+        coordinateSystem(preserveAspectRatio=false)),
+    __Dymola_Commands(file="modelica://Buildings/Resources/Scripts/Dymola/ChillerWSE/Examples/ElectricChillerParallel.mos"
```
Comment (JayHuLBL) [inline Buildings/ChillerWSE/BaseClasses/Examples/BaseClasses/PartialPlantWithControl.mo]: update the script file

File: Buildings/ChillerWSE/BaseClasses/Examples/BaseClasses/PartialPlantWithControl.mo [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+  Fluid.FixedResistances.PressureDrop           res2(
+    redeclare package Medium = Medium2,
+    m_flow_nominal=m2_flow_nominal,
+    dp_nominal=6000)                  "Flow resistance"
+    annotation (Placement(transformation(extent={{-52,-30},{-72,-10}})));
+  Modelica.Blocks.Sources.Constant
+                               TSet(k=273.15 + 12)
+              "Set point for leaving chilled water temperature"
+    annotation (Placement(transformation(extent={{-100,60},{-80,80}})));
+equation
```
Comment (JayHuLBL) [inline Buildings/ChillerWSE/BaseClasses/Examples/BaseClasses/PartialPlantWithControl.mo]: move annotation to new line, more readable.

File: Buildings/ChillerWSE/BaseClasses/FourPortResistanceChillerWSE.mo [hunk 8]
```diff
@@ -0,0 +1,66 @@
+within Buildings.ChillerWSE.BaseClasses;
+record FourPortResistanceChillerWSE
+  "Flow resistance model for the chiller and WSE package"
+
+  parameter Boolean computeFlowResistance1 = true
```
Comment (JayHuLBL) [inline Buildings/ChillerWSE/BaseClasses/FourPortResistanceChillerWSE.mo]: follow naming conventions, `computeFlowResistance1` to `comFloRes1`

File: Buildings/ChillerWSE/BaseClasses/FourPortResistanceChillerWSE.mo [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+    annotation(Dialog(enable = computeFlowResistance1,
+               tab="Flow resistance", group="Medium 1"));
+  parameter Real deltaM1 = 0.1
+    "Fraction of nominal flow rate where flow transitions to laminar"
+    annotation(Dialog(enable = computeFlowResistance1,
+                      tab="Flow resistance", group="Medium 1"));
+  parameter Boolean computeFlowResistance2 = true
+    "=true, compute flow resistance. Set to false to assume no friction"
+    annotation (Evaluate=true, Dialog(tab="Flow resistance", group="Medium 2"));
+
```
Comment (JayHuLBL) [inline Buildings/ChillerWSE/BaseClasses/FourPortResistanceChillerWSE.mo]: delete blank line

File: Buildings/ChillerWSE/BaseClasses/PartialChillerWSE.mo [hunk 10]
```diff
@@ -0,0 +1,297 @@
...
+  Modelica.Blocks.Interfaces.RealOutput wseCHWST(
+    final quantity="ThermodynamicTemperature",
+    final unit="K",
+    displayUnit="degC",
+    min=0,
+    start=T2_start)
+    "Chilled water supply temperature in the waterside economizer" annotation (
+      Placement(transformation(extent={{100,30},{120,50}}), iconTransformation(
+          extent={{100,30},{120,50}})));
+equation
```
Comment (JayHuLBL) [inline Buildings/ChillerWSE/BaseClasses/PartialChillerWSE.mo]: re-format annotation text

File: Buildings/ChillerWSE/BaseClasses/PartialChillerWSE.mo [hunk 11]
```diff
@@ -0,0 +1,297 @@
...
+                                                                 Diagram(
+        coordinateSystem(preserveAspectRatio=false)),
+    Documentation(revisions="<html>
+<ul>
+<li>
+June 30, 2017, by Yangyang Fu:<br/>
+First implementation.
+</li>
+</ul>
+</html>", info="<html>
```
Comment (JayHuLBL) [inline Buildings/ChillerWSE/BaseClasses/PartialChillerWSE.mo]: better move the `info` section before `revisions` section

File: Buildings/ChillerWSE/BaseClasses/PartialChillerWSEInterface.mo [hunk 12]
```diff
@@ -0,0 +1,101 @@
...
+    "= true, if actual temperature at port is computed"
+    annotation(Dialog(tab="Advanced",group="Diagnostics"));
+  parameter Integer n(min=1)=2 "Total number of chillers and waterside economizer";
+
+
+  Modelica.Blocks.Interfaces.RealInput TSet(unit="K", displayUnit="degC")
+    "Set point for leaving water temperature"
+    annotation (Placement(transformation(extent={{-140,84},{-100,124}}),
+        iconTransformation(extent={{-132,92},{-100,124}})));
+  Modelica.Blocks.Interfaces.BooleanInput on[n]
```
Comment (JayHuLBL) [inline Buildings/ChillerWSE/BaseClasses/PartialChillerWSEInterface.mo]: change name: `on` to `uChi`

File: Buildings/ChillerWSE/BaseClasses/Examples/BaseClasses/PartialPlantWithControl.mo (hunk 1 above)
Comment (YangyangFu) [inline Buildings/ChillerWSE/BaseClasses/Examples/BaseClasses/PartialPlantWithControl.mo]: Dr. Hu, @JayHuLBL this model has been deleted in the latest update before I made this pull request. I don't have this model in my current branch. Do you have any idea why it's still in the pull request?

File: Buildings/ChillerWSE/BaseClasses/Examples/BaseClasses/PartialPlantWithControl.mo (hunk 2 above)
Comment (YangyangFu) [inline Buildings/ChillerWSE/BaseClasses/Examples/BaseClasses/PartialPlantWithControl.mo]: I don't have this model either in current branch.

File: Buildings/ChillerWSE/BaseClasses/Examples/BaseClasses/PartialPlantWithControl.mo (hunk 3 above)
Comment (YangyangFu) [inline Buildings/ChillerWSE/BaseClasses/Examples/BaseClasses/PartialPlantWithControl.mo]: This model is deleted.

File: Buildings/ChillerWSE/BaseClasses/Examples/BaseClasses/PartialPlantWithControl.mo (hunk 1 above)
Comment (JayHuLBL) [inline Buildings/ChillerWSE/BaseClasses/Examples/BaseClasses/PartialPlantWithControl.mo]: @YangyangFu You may have deleted the `Buildings.ChillerWSE.BaseClasses.Examples` folder using Dymola and it did not show up when opening the Building library in Dymola. But the file folder `modelica-buildings/Buildings/ChillerWSE/BaseClasses/Examples/` is still in your branch. You should manually delete the folder.

File: Buildings/ChillerWSE/BaseClasses/FourPortResistanceChillerWSE.mo (hunk 8 above)
Comment (YangyangFu) [inline Buildings/ChillerWSE/BaseClasses/FourPortResistanceChillerWSE.mo]: Dr. Wetter used this naming in most models.

**Pull request merged**